app/components/UI/Snaps/SnapUIImage/SnapUIImage.tsx

```diff
@@ -8,7 +8,7 @@ export interface SnapUIImageProps {
   style?: StyleProp<ImageStyle>;
   width?: number;
   height?: number;
-  borderRadius?: number;
+  borderRadius?: number | string;
```

Comment:
```suggestion
  borderRadius?: number | 'full';
```
Would the TS checks be simplified by this?